<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label3_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 68, 97);</string>
   </property>
   <property name="text">
    <string>BOTTLE INSPECTION SYSTEM</string>
   </property>
  </widget>
  <widget class="QLabel" name="label4_4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>174</y>
     <width>47</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>MAIL ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1_1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/newPrefix/spark-drives.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic.qrc">:/newPrefix/spark-drives.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>134</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label5_5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>134</y>
     <width>91</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>YOUR ID</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>-30</y>
     <width>20</width>
     <height>511</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_1">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>174</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label2_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Arial&quot;;
COLOR:rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>SPARK DRIVES AND AUTOMATION</string>
   </property>
  </widget>
  <widget class="QLabel" name="label5_7">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>213</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>ORGANIZATION</string>
   </property>
  </widget>
  <widget class="QLabel" name="label6_6">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>253</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>PASSWORD</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit2_3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>213</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit2_4">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>253</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton2_1">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>SIGN UP</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>293</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Admin</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>293</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Employee</string>
   </property>
  </widget>
  <widget class="QLabel" name="label5_6">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>293</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>PROFILE</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
